runtime/kusama/src/lib.rs

```diff
@@ -410,11 +410,7 @@ parameter_types! {
 
 	// 1 hour session, 15 minutes unsigned phase, 8 offchain executions.
 	pub OffchainRepeat: BlockNumber = UnsignedPhase::get() / 8;
-
-	/// Whilst `UseNominatorsAndUpdateBagsList` or `UseNominatorsMap` is in use, this can still be a
-	/// very large value. Once the `BagsList` is in full motion, staking might open its door to many
-	/// more nominators, and this value should instead be what is a "safe" number (e.g. 22500).
-	pub const VoterSnapshotPerBlock: u32 = 22_500;
+	pub const VoterSnapshotPerBlock: u32 = 12_500;
```

Comment:
But y u remove comment?

Reply:
comment was referring to the pre-bags-list world, we are passed that now :)

Reply:
Yeah but what I mean was, it should still have a comment explaining why we have set it to be `12_500`.